shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/instance/metadata/jdbc/JDBCInstanceMetaData.java

```diff
@@ -32,9 +32,18 @@ public final class JDBCInstanceMetaData implements InstanceMetaData {
     
     private final String ip;
     
+    private final boolean force;
```

Comment:
@zhaojinchao95 Can you help review this change?

Reply:
I think the `force` parameter move to props is better.